packages/widgets/src/media-server/component.tsx

```diff
@@ -83,49 +90,141 @@ export default function MediaServerWidget({ integrationIds, isEditMode }: Widget
 
   // Only render the flat list of sessions when the currentStreams change
   // Otherwise it will always create a new array reference and cause the table to re-render
-  const flatSessions = useMemo(() => currentStreams.flatMap((pair) => pair.sessions), [currentStreams]);
+  const flatSessions = useMemo(
+    () =>
+      currentStreams.flatMap((pair) =>
+        pair.sessions.map((session) => ({
+          ...session,
+          integrationKind: pair.integrationKind,
+          integrationName: integrationDefs[pair.integrationKind].name,
+          integrationIcon: getIconUrl(pair.integrationKind),
+        })),
+      ),
+    [currentStreams],
+  );
 
+  const baseStyle: MantineStyleProp = {
+    "--total-width": "calc(100cqw / var(--total-width))",
+    "--ratio-width": "calc(100cqw / var(--total-width))",
+    "--space-size": "calc(var(--ratio-width) * 0.1)", //Standard gap and spacing value
+    "--text-fz": "calc(var(--ratio-width) * 0.45)", //General Font Size
+    "--mrt-base-background-color": "transparent",
+  };
+  const { openModal } = useModalAction(itemInfoModal);
   const table = useTranslatedMantineReactTable({
     columns,
     data: flatSessions,
-    enableRowSelection: false,
+    enablePagination: false,
+    enableTopToolbar: false,
+    enableBottomToolbar: false,
+    enableSorting: false,
+    enableColumnActions: false,
+    enableStickyHeader: false,
     enableColumnOrdering: false,
+    enableRowSelection: false,
     enableFullScreenToggle: false,
     enableGlobalFilter: false,
     enableDensityToggle: false,
     enableFilters: false,
-    enablePagination: true,
-    enableSorting: true,
     enableHiding: false,
-    enableTopToolbar: false,
-    enableColumnActions: false,
-    enableStickyHeader: true,
     initialState: {
       density: "xs",
     },
     mantinePaperProps: {
-      display: "flex",
-      h: "100%",
+      flex: 1,
       withBorder: false,
-      style: {
-        flexDirection: "column",
-      },
+      shadow: undefined,
     },
     mantineTableProps: {
+      className: "media-server-widget-table",
       style: {
         tableLayout: "fixed",
       },
     },
     mantineTableContainerProps: {
       style: {
-        flexGrow: 5,
+        height: "100%",
       },
     },
+    mantineTableBodyCellProps: ({ row }) => ({
+      onClick: () => {
+        openModal({
+          item: row.original,
+        });
```

Comment:
You probably want to add the custom title or not?

Reply:
Also what happened to the icon?

Reply:
I want that the title will be the icon but it can't be done. do you have a better idea?